src/profile-logic/import/instruments/index.js

```diff
@@ -0,0 +1,899 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+// @flow
+
+//types
+import type {
+  Profile,
+  FuncTable,
+  FrameTable,
+  StackTable,
+} from '../../../types/profile';
+
+type Sample = {|
```

Comment:
I think it would be better to rename this to `InstrumentsSample` since we have a `Sample` type already. That can confuse people.